docs/source/whitepaper.rst

```diff
@@ -391,7 +391,7 @@ There are two fees that users need to pay to bridge their tokens:
 
 agent fee
     The fee paid in token being moved, covering the gas costs and rewarding
-    the liquidity provider. Variable and currently set to ``max(5e18, 0.1% of
+    the liquidity provider. Variable and currently set to ``max(1e18, 0.3% of
```

Comment:
1e18 is misleading in the context of USDC. But I don't have a better idea to represent the value, independently of the number of decimals. (i.e. for usdc it's 1e6)

Reply:
maybe 1e[NUMBER_OF_DECIMALS]

Reply:
Yeah, I was wondering what to do here too...

Reply:
Expressed the min LP fee in terms of `token.decimals()`. Also slightly adjusted the presentation.